### PR TITLE
Add scripts and config for deploying TACo application to dashboard do…

### PR DESCRIPTION
**Pull request by theref**

### CODE REVIEW

**Comment by KPrasch** [comment]
Looking great 👍🏻 Shall we make this deployment upgradable so that we can mitigate any future need for testnet application authorizations?

File: deployment/constructor_params/dashboard/root.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    filename: dashboard.json
+
+constants:
+   IN_SECONDS_1_HOUR: 3600
+   IN_SECONDS_7_DAYS: 604800
+   IN_SECONDS_60_DAYS: 5184000
+   IN_SECONDS_182_DAYS: 15724800
+   IN_SECONDS_364_DAYS: 31449600
+   FORTY_THOUSAND_TOKENS_IN_WEI_UNITS: 40000000000000000000000  
+   TEN_MILLION_TOKENS_IN_WEI_UNITS: 10000000000000000000000000
```
Comment (derekpierre) [inline deployment/constructor_params/dashboard/root.yml:16]: Don't need this constant.

```suggestion
```

File: deployment/constructor_params/dashboard/root.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+contracts:
+  - TACoApplication:
+      _token: "0x3f16380656cAE45D3f80D8833682d2b606eD094A"
+      _tStaking: "0x1da5d88C26EA4f87b5e09C3452eE2384Ee20DC75"
```
Comment (derekpierre) [inline deployment/constructor_params/dashboard/root.yml:21]: 👍 💥

**Review by derekpierre — APPROVED** [review]
🎸  - Good stuff! One suggestion to remove an unnecessary constant.

**Comment by derekpierre** [comment]
Do you plan on updating the dashboard.json after the actual goerli deployment in this PR or a separate one?

File: deployment/constructor_params/dashboard/root.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+deployment:
+  name: dashboard-root
+  chain_id: 5
+
+artifacts:
+    dir: ./deployment/artifacts/
+    filename: dashboard.json
```
[the remark this answers is not in the document]
Comment (theref) [reply deployment/constructor_params/dashboard/root.yml:7]: didn't happen

**Review by KPrasch — APPROVED** [review]

**Pull request merged**